src/book/02-system/03-tools/06-natnet.mdx

```diff
@@ -0,0 +1,104 @@
+---
+section: System
+chapter: Tools
+title: NatNet SDK
+description: A tool that provides positional infomation from motive mocap software.
+slug: /system/tools/natnet_sdk
+authors:
+  - Luke Haigh
+---
+
+NatNet is module that works in conjunction with the [OptiTrack Motive software](https://www.optitrack.com/software/motive/) and motion tracking camera system. The purpose of the NatNat system is to record positional information about the NUgus robots in relation to the playing field and the external envrionment so that it can be provided to our automatic anotation system which will use the positional information of the robots to automatically create segmentation masks of the environment. 
+
+## Getting NatNet Running
+
+In order to get NatNet running you will require a licenced version of the Optitrack Motive software and at least 2 [Optive Motion Capture Cameras](https://www.optitrack.com/cameras/). NatNet currently supports motive version 3.0 and below. Below is a detailed guide on how to setup motive.
+
+<details>
+<summary>Setting up Motive</summary>
+
+1. Follow the [Optitrack guide for Installation and Activation](https://docs.optitrack.com/motive/installation-and-activation) if you don't already have Motive installed.
+
+2. Open the Motive software. You will be presented with a range of options, but it is safest to choose "Perform Camera Calibration" and re-calibrate the cameras, in case one or two of them have been accidentally moved since the last calibration.
+
+3. You should have 10 black boxes visible at the bottom of your screen representing the view of the 10 cameras mounted around the room (they will default to object mode which only displays objects whiter than a certain threshold, assuming them to be motion capture dots. If you don't have 10 camera boxes displaying, check the netgear ethernet switch is powered on. If the cameras are on, they will display a number on their LCD screen.
```

Comment:
probably specify somewhere that this is for the lab. or instead of ten, just say 'the number of cameras you're using'.